clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -100,6 +100,16 @@
                     "uniqueItems": true
                 }
             }
+        },
+        "configurationDefaults": {
+            "[COBOL]": {
+                "editor.renderIndentGuides": false,
+                "editor.rulers": [
+                    6,
+                    7,
```

Comment:
Should we add also 11 here to show Area B?

Reply:
Make sense. I could also propose 80 for showing the end of the line.
Do you think that we need it as well?

Reply:
It is a UX question, we should agree it with Lukas